### PR TITLE
fix(docs): updates a few missing types from buckets/threads

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: etc/hub.api.md [hunk 1]
```diff
@@ -909,13 +908,14 @@ export const Where: typeof Criterion;
 
 // @public
 export interface WithKeyInfoOptions extends WithUserAuthOptions {
-    // (undocumented)
     date?: Date;
+    debug?: boolean;
+    host?: string;
```
Comment (deefactorial) [inline etc/hub.api.md:913]: Isn't the `host` and `debug` properties implied if it extends the `WithUserAuthOptions` type with those properties in it? Why the nested extensions if they are duplicated?

File: packages/grpc-authentication/src/index.ts [hunk 2]
```diff
@@ -8,20 +8,41 @@ import { Client } from '@textile/hub-threads-client'
...
+   */
+  debug?: boolean
+  /**
+   * Optional. Future date for the session to expire. Short time periods are strongly recommended. (Default 30 minutes).
+   */
   date?: Date
+  /**
+   * Development purposes only.
+   */
+  host?: string
```
Comment (deefactorial) [inline packages/grpc-authentication/src/index.ts:22]: same here with `host` and `debug` properties being implied

**Review by deefactorial — COMMENTED** [review]
Questions

File: etc/hub.api.md (hunk 1 above)
Comment (andrewxhill) [reply etc/hub.api.md:913]: it's all about the static docs generation. unfortunately, our generator doesn't pick up the implied properties.

File: packages/grpc-authentication/src/index.ts (hunk 2 above)
Comment (andrewxhill) [reply packages/grpc-authentication/src/index.ts:22]: same yeah

**Review by carsonfarmer — APPROVED** [review]
Ét voila!

**Pull request merged**